src/lib.rs

```diff
@@ -71,13 +66,11 @@ impl<R: Runtime> ContextMenu<R> {
     #[cfg(target_os = "linux")]
```

Comment:
The two show_context_menu functions (for linux and for windows-mac) can probably be merged in a single function

Reply:
Will be done in a separate PR